### PR TITLE
configure nc ionos processes

### PR DESCRIPTION
**Pull request by printminion-co**
in order to configure nc_ionos_processes app

should be merged right before https://github.com/IONOS-Productivity/nc-server/pull/59

### CODE REVIEW

File: configure.sh [hunk 1]
```diff
@@ -59,6 +59,18 @@ config_apps() {
...
 	ooc config:app:set --value='["admin"]' core shareapi_only_share_with_group_members_exclude_group_list
+
+	echo "Configure nc_ionos_processes app"
+
+	if [ -z "${IONOS_PROCESSES_API_URL}" ] || [ -z "${IONOS_PROCESSES_USER}" ] || [ -z "${IONOS_PROCESSES_PASS}" ]; then
+		echo "Warning: IONOS_PROCESSES_API_URL, IONOS_PROCESSES_USER or IONOS_PROCESSES_PASS not set, skipping configuration of nc_ionos_processes app"
+		return
+	fi
+
+	ooc app:enable nc_ionos_processes
```
Comment (thlehmann-ionos) [inline configure.sh:70]: Enabling this app is already done in `apps-enable.sh`

Comment (printminion-co) [reply configure.sh]: removed

**Review by thlehmann-ionos — CHANGES_REQUESTED** [review]
Review OK if you've removed the redundant app enabling.

**Review by thlehmann-ionos — APPROVED** [review]
LGTM.

**Pull request merged**